tests/unit/NumberUtilsTest.js

```diff
@@ -0,0 +1,14 @@
+const NumberUtils = require('../../src/libs/NumberUtils');
+
+const result = NumberUtils.rand64();
+
+const max64 = 9223372036854775807;
```

Comment:
How does this work? As far as I know, this will result in a floating point number with a loss of precision.

Reply:
Yeah, I think this needs to be a BigInt

Reply:
it should be a string, no? since we are generating a numeric string id